origami.py

```diff
@@ -309,11 +309,20 @@ def _on_resize_panes(self, orientation, cells, relevant_index, orig_data, text):
         layout = self._on_resize_panes_get_layout(orientation, cells, relevant_index, orig_data, text)
         self.window.set_layout(layout)
 
-    def zoom_pane(self, fraction):
-        if fraction is None:
-            fraction = .9
+    
+
+    def zoom_pane(self, auto_zoom):
+        fraction_horizontal = .9
+        fraction_vertical = .9
+        
+        if isinstance(auto_zoom, float) or isinstance(auto_zoom, int):
+            fraction_horizontal = auto_zoom
```

Comment:
I don't know clear your mean. Why to set both fractions？

Reply:
Didn't the original code apply specified fraction to both width and height? Here you are applying it only to width, leaving the default `.9` to be used for height.

Reply:
This is what I'm talking about:

```suggestion
            fraction_horizontal = fraction_vertical = auto_zoom
```

since a single value was previously applying both in horizontal and vertical cases.

(and please let me resolve the issues once I verify that it was addressed)